lua/cfcfactions/core/cl/dermas/cl_usersderma.lua

```diff
@@ -1,8 +1,24 @@
 if not CLIENT then return end
 
+local cfcusers = {}
+
 local Panel = {}
```

Comment:
```suggestion
local ViewUserDerma = {}
```
Or something like that?
@brandonsturgeon ideas?